python-avd/pyavd/_eos_designs/schema/schema_fragments/digital_twin.schema.yml

```
@@ -48,6 +48,12 @@ keys:
               Specifies if the ACT Digital Twin device is deployed with direct access to the Internet.
               This option applies only to the 'cloudeos' and 'veos' node types and will be ignored for all other ACT node types.
               ACT does not provide direct Internet access to cloudeos or veos devices by default.
+      default_interfaces_of_original_platform:
+        type: bool
+        default: false
+        description: |-
+           In Digital Twin mode, AVD by default uses the `default_interfaces` settings associated with the alternate platform (identified via `platform_settings.[].digital_twin.platform`).
+           Set key `digital_twin.default_interfaces_of_original_platform` to `True` to use `default_interfaces` settings of the original platform.
```

Let's try to make the key name as intuitive as possible so the description can be shorter and the brain load lower :)
```suggestion
           In Digital Twin mode, AVD can either use the default interfaces of the original or the digital twin platform (as set in `platform_settings.[].digital_twin.platform`).
```

---

Changed to suggested description

---

We might need to update this if we change back the key per my comment above.